crates/agent/src/thread.rs

```diff
@@ -1389,7 +1389,7 @@ impl Thread {
         use LanguageModelCompletionEvent::*;
 
         match event {
-            StartMessage { .. } => {
+            LanguageModelCompletionEvent::StartMessage { .. } => {
```

Comment:
```suggestion
            StartMessage { .. } => {
```

for stylistic consistency with the other patterns!